src/projects/TodoList/index.js

```diff
@@ -1,17 +1,40 @@
 import React from 'react';
 import style from './style.css';
 
-class TodoList extends React.Component {
-
+class Todo extends React.Component {
   render() {
-    /*
-      Before the return you can do all logic you need
-    */
     return (
-      <p>Hey im a todo list</p>
+      <p>{this.props.eachTodo}</p>
     )
   }
+}
 
+class TodoList extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      todos: ['Take out the bins', 'Put a wash load on', 'Learn React', 'Gitgud at Overwatch'],
+      test: 'Tester'
+    }
+  }
+  render() {
+    // Before the return you can do all logic you need
+    let eachTodo = this.state.todos.map((item) => {
+      return (
+        <Todo
+          eachTodo={item}
```

Comment:
Not sure if I like the prop name `eachTodo` - something as simple as `todo` or `task` is a bit more descriptive imo. Remember this is an individual instance of a `<Todo />` component so having the word `each` makes it seem like its not an individual instance (if that makes sense)

Reply:
Don't forget to change it in the Todo component as well if you change it here